src/lib/journey/callbacks/recaptcha/recaptcha.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import type { ReCaptchaCallback } from '@forgerock/javascript-sdk';
+  import type { z } from 'zod';
+  import { onMount } from 'svelte';
+
+  import type { SelfSubmitFunction, StepMetadata } from '$journey/journey.interfaces';
+  import type { styleSchema } from '$lib/style.store';
+  import type { Maybe } from '$lib/interfaces';
+
+  export let callback: Maybe<ReCaptchaCallback>;
+  export const selfSubmitFunction: Maybe<SelfSubmitFunction> = null;
+  export const stepMetadata: Maybe<StepMetadata> = null;
+  export const style: z.infer<typeof styleSchema> = {};
+
+  const siteKey = callback?.getSiteKey() ?? '';
+  let recaptchaApi = `${callback?.getOutputByName<string>('captchaApiUri', '') ?? ''}`;
+  /**
+   * AM defaults the class name to g-captcha which is wrong
+   * I dont think we should be manipulating the class here,
+   * but the classname should be g-recaptcha for google
+   */
+  const recaptchaClass: string =
```

Comment:
By "class" here, I'm assuming we mean CSS class, yeah? And, this is a reference to the element to which the reCaptcha is expected to mount? I see we are using that class on the element template below, so the Captcha will mount to this component's HTML?

Reply:
Correct, the class is the css class, and its the class which hcaptcha or grecaptcha is going to attempt to inject its iframe into.